CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to Paywall
+
+We want to make contributing to this project as easy and transparent as
+possible, and actively welcome your pull requests. If you run into problems,
+please open an issue on GitHub.
+
+## Pull Requests
+
+1. Fork the repo and create your branch from `develop`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints (see below).
+6. Tag @yusuftor, @jakemore, and @anglinb in the pull request.
+
+## Coding Style
+
+### Xcode Preferences
+In Xcode, you'll need to set your indentation spacing to 2 spaces. You can do that by going to **Xcode ▸ Preferences...**, then choose the **Text Editing** tab. Inside the **Display** tab, set **Indent wrapped lines by:** to **2 spaces**:
+
+<p align="center">
+  <img width="830" alt="Indent wrapped lines by 2 spaces" src="https://user-images.githubusercontent.com/3296904/159707444-c3524fef-b31f-4495-b8ca-2ecd8fc7f698.png">
+</p>
+
+In the **Editing** tab, select **Automatically trim trailing whitespace**:
+
+<p align="center">
+  <img width="830" alt="Automatically trim trailing whitespace" src="https://user-images.githubusercontent.com/3296904/159707570-5e38f179-7568-4a49-89b5-25158686e439.png">
+</p>
+
+In the **Indentation** tab, set **Tab Width** and **Indent Width** to **2 spaces**:
+
+<p align="center">
+  <img width="830" alt="2 space indent and tab width" src="https://user-images.githubusercontent.com/3296904/159707661-46e1eed0-d90e-4912-af8c-28e987d5ab60.png">
+</p>
+
+### SwiftLint
```

Comment:
Yeah SwiftLint doesn't work with packages. You can run it in terminal. There was some talk about someone might create a plugin for SwiftLint to use with packages with Xcode 13.3, but that hasn't happened yet. In this contributing.md it's basically just saying rely on GitHub actions for now

Reply:
I saw that link you posted before, but I thought it might be annoying to include because it creates a text file every time with all the errors/warnings